source/change-streams/change-streams.rst

```diff
@@ -278,6 +278,16 @@ Driver API
      * @note this is an option of the `$changeStream` pipeline stage.
      */
     startAtOperationTime: Optional<Timestamp>;
+
+    /**
+     * Similar to 'resumeAfter', this option takes a resume token and starts a
+     * new change stream returning the first notification after the token.
+     * This will allow users to watch collections that have been dropped and recreated
+     * or newly renamed collections without missing any notifications.
```

Comment:
Should we mention that this option is mutually exclusive with `resumeAfter`? If you look at `document_source_change_stream.cpp` in the code review for [SERVER-35030](https://jira.mongodb.org/browse/SERVER-35030) you can see that the server enforces that. I don't think drivers need to add client-side validation, but it may be helpful to document this with a line like:

> The server will report an error if ``startAfter`` and ``resumeAfter`` are both specified.